vms/platformvm/upgrade/config.go

```diff
@@ -0,0 +1,50 @@
+// Copyright (C) 2019-2024, Ava Labs, Inc. All rights reserved.
+// See the file LICENSE for licensing terms.
+
+package upgrade
+
+import "time"
+
+type Config struct {
```

Comment:
@joshua-kim maybe `Upgrades` instead? I'll be more readable when we use it in `vm.Config`

Reply:
I think the code should make sense in context of the packages - `upgrade.Config` i think makes sense within this package's context and if we're worried about it being confusing in `vm.Config` we don't have to embed and we can just name it something like `UpgradeConfig` or `ForkConfig`